controller/search.js

```diff
@@ -110,7 +113,7 @@ function setup( apiConfig, esclient, query, should_execute ){
         logger.debug('[ES response]', docs);
         next();
       });
-      debugLog.stopTimer(req);
+      debugLog.stopTimer(req, initialTime);
```

Comment:
Same comments as with controllers/place, `initialTime` never changes and therefore reports incorrect times.